parsec/core/mountpoint/winfsp_operations.py

```diff
@@ -144,13 +146,14 @@ def get_path_and_translate_error(
         raise NTStatusError(NTSTATUS.STATUS_INTERNAL_ERROR) from exc
 
 
-def round_to_block_size(size, block_size=DEFAULT_BLOCK_SIZE):
+def round_to_block_size(size: int, block_size: int = DEFAULT_BLOCK_SIZE) -> int:
     if size % block_size == 0:
         return size
     return ((size // block_size) + 1) * block_size
 
 
-def stat_to_file_attributes(stat):
+# winfsp is present only in a windows env, so we ignore this type
```

Comment:
Which type error ?

Reply:
I removed this comment, it's not needed anymore

Reply:
:+1: